shopping-cart/shopping-cart.js

```diff
@@ -0,0 +1,18 @@
+import cart from '../data/cart.js';
+import cameras from '../data/cameras.js';
+import { findById, calcOrderTotal, toUSD } from '../common/utils.js';
+import renderLineItem from '../shopping-cart/generate-line-items.js';
+
+const tbody = document.querySelector('tbody');
+const orderTotalCell = document.getElementById('subtotal-cell');
+
+for (let i = 0; i < cart.length; i++) {
+    const lineItem = cart[i];
+    const cameras = findById(cameras, lineItem.id);
```

Comment:
You need a different name for your variable in this line - it gives the error:
 Uncaught ReferenceError: Cannot access 'cameras' before initialization
because it thinks the cameras in the argument is the cameras that you're declaring in this line. It'll also cause issues because you import cameras on line 2